frontend/src/main/components/Users/UsersTable.js

```diff
@@ -15,18 +16,32 @@ export default function UsersTable({ users}) {
             }
         }
     }
+    function cellToAxiosParamsToggleDriver(cell) {
+        return {
+            url: "/api/admin/users/toggleDriver",
+            method: "POST",
+            params: {
+                id: cell.row.values.id
+            }
+        }
+    }
 
     // Stryker disable all : hard to test for query caching
     const toggleAdminMutation = useBackendMutation(
         cellToAxiosParamsToggleAdmin,
         {},
         ["/api/admin/users"]
     );
+    const toggleDriverMutation = useBackendMutation(
+        cellToAxiosParamsToggleDriver,
+        {},
+        ["/api/admin/users"]
+    );
     // Stryker enable all 
```

Comment:
Should be 
```suggestion
    // Stryker restore all 
```

See: https://ucsb-cs156-s23.slack.com/archives/C050779M19P/p1685726928710669